src/server/Document/attributeFunctions.js

```diff
@@ -0,0 +1,25 @@
+import encodeChunkFilename from '../utilities/encodeChunkUri';
```

Comment:
This might be a personal opinion but the naming of this file is a little odd. We don't have the naming `{something}Functions` anywhere else in the codebase. If it were me I'd name this file `utils.js` or separate this file into 2 files `getScriptAttributes.js` and `getLinkAttributes.js` and you could even pop them into a `utils` folder but not really necessary.

Reply:
I named them 'attribute functions' to reflect what their typescript types describe their functions as, `AttrFn`:
![image](https://user-images.githubusercontent.com/9645462/148513686-8f11adde-da26-4327-a60d-5f8adcbc41e3.png)
I'd be happy to move this to utilities.

Reply:
Ah ok. That's up to you. Just worried about `{something}Functions` spreading in our codebase if someone thinks it's a convention we have

Reply:
Moved https://github.com/bbc/simorgh/pull/9771/commits/3846a569d0fcb04fb0a47918e421f6990c50d3b4

Reply:
I still find the naming of a file `{something}Functions` odd but this is probably more of a personal opinion. Thanks for addressing